courses/templates/course_detail.html

```diff
@@ -80,4 +80,26 @@ <h2>{{ course.subhead }}</h2>
         </div>
     </div>
 </div>
+{% if course.for_teams %}
+  <div class="for-teams-block">
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-7 {% if course.for_teams.switch_layout %}order-2{% endif %}">
+          <h1>For Teams</h1>
+          {{ course.for_teams.content|safe }}
+          <a href="#" class="btn btn-primary text-uppercase px-5 py-2 action-button">{{ course.for_teams.action_title }}</a>
```

Comment:
why we making this `<a>`?

Reply:
Can be switched to a `<button>` if required. Behavior is not defined yet.